extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -120,28 +124,39 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 
   /**
-   * The listener function that listen on onScroll event and
-   * show sticky ad when user scroll at least one viewport and
-   * there is at least one more viewport available.
+   * The listener function that listen on viewport scroll event.
+   * And decide when to display the ad.
    * @private
    */
-  displayAfterScroll_() {
+  onScroll_() {
+    if (isExperimentOn(this.win, TAG)) {
+      this.displayAfterScroll_();
```

Comment:
This would show it as soon as scrolling starts, we want to show the ad when scrolling *ends*,  `resources-impl` already has some logic for detecting scroll stop, it would be nice to refactor that into a common helper to use here as well. I think a `var scrollEndUnlisten = this.viewport_.onScrollEnd()` would be a nice API to have that can be used by this and `resources-impl`